docs/api-reference/static-analysis.md

```diff
@@ -40,109 +45,219 @@ npm install --save-dev @dbos-inc/eslint-plugin
+            globals: { ...globals.node, ...globals.es6 }
+        },
+
+        rules: { }
+      }
+    );
+    ```
+  </TabItem>
+  <TabItem value="legacy-config" label="Legacy config">
+    <h4> This config style will work with ESLint 8 and below. </h4>
+    Place an `.eslintrc` file similar to the following in your project directory:
+
+    ```json
+    {
+      "root": true,
+      "extends": [
+        "plugin:@dbos-inc/dbosRecommendedConfig"
+      ],
+      "plugins": [
+        "@dbos-inc"
+      ],
+      "env": {
+        "node": true,
+        "es6": true
+      },
+      "rules": {
+      },
+      "parser": "@typescript-eslint/parser",
+      "parserOptions": {
+        "project": "./tsconfig.json"
+      }
+    }
+    ```
+  </TabItem>
+</Tabs>
 
 The example above configures the project for the recommended `eslint` configuration.  Adjust the `extends`, `rules`, `plugins`, `env`, and other sections as desired, consulting the configurations and rules below.
 
 Finally, to make `eslint` easy to run, it is suggested to place commands in `package.json`.  For example:
-```json
-  "scripts": {
-    "build": "tsc",
-    "test": "...",
-    "lint": "eslint src",
-    "lint-fix": "eslint --fix src"
-  },
 
+```json
+"scripts": {
+  "build": "tsc",
+  "test": "...",
+  "lint": "eslint src",
+  "lint-fix": "eslint --fix src"
+}
 ```
 
 #### Profiles to extend
 
-* `dbosBaseConfig` - This configuration includes only rules required for security.  These rules will be enforced prior to deployment to the DBOS cloud.
-* `dbosRecommendedConfig` - This extension to the base configuration is the recommended configuration, which includes best practices for new TypeScript code.
-* `dbosExtendedConfig` - This configuration extenends the recommended configuration, and may include opinions on coding style.
-
-### Suggested `eslint` Rules
-
-#### `no-console`
-Many programmers use `console` statements during the development and debugging process.  However, `console` is not suitable for production usage.
+* `dbosRecommendedConfig`: this extension to the base configuration is the recommended configuration, which includes best practices for new TypeScript code.
+* `dbosBaseConfig`, `dbosExtendedConfig`: these configurations are deprecated, and are now the same as `dbosRecommendedConfig`.
 
-The `no-console` rule, provided by the primary `eslint` package, is automatically enabled in all configs of the DBOS plugin.
+---
 
-Use of `console` should be removed from code prior to use in production.  Activities such as logging should use [DBOS Transact logging facilities](../tutorials/logging.md), so that data remains structured and can be automatically collected in a central database.
+### Recommended rules/plugins
 
-#### `no-eval`
-Interpreted languages like JavaScript support the ability to treat data directly as executable code.  If arbitrary user data can become code, many of the negative security implications are obvious.
+These rules are added:
```

Comment:
It's more like "These rules are enabled by default"

Reply:
Got it